tests/test_acpi.sh

```diff
@@ -58,15 +56,14 @@ temp_outputs+=(
 )
 temp_values+=("")
 
-function test_acpi_battery {
+test_acpi_battery() {
```

Comment:
No need to make this change on this line.

Reply:
That was left by accident. Actually, on my machine, I have to change to `function_name()` for tests to run, if it's `function function_name` it just doesn't run any tests.